### ISSUE TITLE
Update words in camera input preferences

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#491]
Over in https://github.com/phetsims/ratio-and-proportion/issues/431#issuecomment-1206666577 we are now calling things "Camera Input: Hands"

Here are the current preferences:

![image](https://user-images.githubusercontent.com/6856943/183132793-7b152679-dc4c-4b8b-a1ee-1cba67f359d3.png)

Over to @terracoda to design this, and potentially add a sentence/description for each. (don't forget to add description).

**Comment by terracoda** [comment]
Remove heading "Sim-specific settings".

**Question:** Should this feature eventually exist under the Input Tab? Maybe only once it can be turned on through the Input Tab. Until then, we need to explain the query parameter and the X-Y settings on the General Tab. 

So assuming we stick the General Tab for now, here are some potential words to describe this featured input method when it is enabled:

**Camera Input: Hands**
Using the query parameter, "cameraInput=hands", your computer's camera recognizes actual hands as input devices. Move hands within the camera's view to move the sim's hands up and down or forward or backward depending on the camera's orientation. 

---

**Camera Input: Hands**
Using the query parameter, "cameraInput=hands", real-world hands are recognized as input devices when moved within the camera's field of view. 

Optimize hand operation with desired camera perspective.
[ ] Flip hand operation from directly mirrored (default) to controlling opposite hand
[ ] Flip camera perspective from frontal view of hands (default) to horizontal view above or below the hands 

---

**Camera Input: Hands**
With "cameraInput=hands" enabled, real-world hands are recognized as input devices and move the sim's Left and Right Hands. 

---

**Camera Input: Hands**
With "cameraInput=hands" enabled, real-world hands move the sim's Left and Right Hands when within the camera's field of view. 

---

**Camera Input: Hands**
With "cameraInput=hands" enabled, real-world hands move the sim's Left and Right Hands. 

Choose camera perspective and orientation for hand operation.
[ ] Use frontal camera view of hands or horizontal view of hands
[ ] Use directly mirrored hand control or opposite (or flipped) hand control

---

Choose camera perspective and orientation for hand operation.
[ ] Flip camera view of hands between frontal and horizontal views
 - Special context responses (no repeat of name response)
     - When checked: Using horizontal view of hands.
     - When unchecked: Using frontal view of hands.

[ ] Flip hand operation between mirrored hand control and opposite hand control.
 - Special context responses (no repeat of name response)
     - When checked: Opposite hand controls sim hand.
     - When unchecked: Hand operation directly mirrored.

**Comment by terracoda** [comment]
I think this seems best...

**Camera Input: Hands**
With "cameraInput=hands" enabled, real-world hands move the sim's Left and Right Hands. 

Choose camera perspective and orientation for hand operation.
[ ] Flip camera view of hands between frontal and horizontal views
 - Special context responses (no repeat of name response)
     - When checked: Using horizontal view of hands.
     - When unchecked: Using frontal view of hands.

[ ] Flip hand operation between mirrored hand control and opposite hand control.
 - Special context responses (no repeat of name response)
     - When checked: Using opposite hand control.
     - When unchecked: Using mirrored hand control.

**Comment by terracoda** [comment]
@zepumph this would mean the dialog content would look something like this:

**Camera Input: Hands**
With "cameraInput=hands" enabled, real-world hands move the sim's Left and Right Hands. 

Choose camera perspective and orientation for hand operation.
[ ] Flip camera view of hands between frontal and horizontal views
[ ] Flip hand operation between mirrored hand control and opposite hand control.

**Comment by terracoda** [comment]
And the reading block for the first part would be:
"Camera Input: Hands. With cameraInput equals hands enabled, real-world hands move the sim's hands Left and Right Hands."

The heading is read out as part of the reading block when focus lands on the reading block below the heading.

**Comment by terracoda** [comment]
Note that the checkboxes have the special checkbox "Preferences Menu" context response pattern where the object name is not repeated with the context response.

**Comment by emily-phet** [comment]
@terracoda Thoughts:
- Yes - this would be perfect for an "input" tab. Let's just have an Input tab for this sim, if that's not a big challenge. 
- Can we use "**use your hands** to move the sim's Left and Right hands" instead of "real world hands"?
- Can we use "Customize hand movement" (instead of "choose camera perspective and orientation for hand operation")

I think the intent of the checkboxes might be best conveyed with a graphic. Can we put an image in an "Input" tab?
Then we could have one (drawn) image of human hands and the sim, with one hand raised and the corresponding hand raised in the sim (with an arrow indicating the motion for the human hand and the sim hand).
Same for the y-axis flip, we could have someone moving their hand perpendicular to the sim and the corresponding sim hand moving upward. 

Then the Voiced / interactive description description could provide the description of the control.

**Comment by terracoda** [comment]
"Yes - this would be perfect for an "input" tab. Let's just have an Input tab for this sim, if that's not a big challenge."

@zepumph, informed me already that doing sim-specific stuff inside the tabs is really hard at the moment.


Re-wording changes...I'll copy out your simplified suggestions. I had a lot of trouble getting my head around the X & Y axes.

**Camera Input: Hands**
With "cameraInput=hands" use your hands to move the sim's Left and Right Hands.

Customize hand movement.
[ ] Flip camera view of hands between frontal and horizontal views
[ ] Flip hand operation between mirrored hand control and opposite hand control

I'll ask about artwork. That's not something I can do.

**Comment by terracoda** [comment]
I'll put the artwork request in a separate issue request.

**Comment by terracoda** [comment]
@zepumph can you update the wording. Then we can decide on which Tab it will go under and what graphics we might need.

**Comment by terracoda** [comment]
Just retyping the content and the response...

**Camera Input: Hands**
With "cameraInput=hands" use your hands to move the sim's Left and Right Hands.

**Customize hand movement**
[ ] Flip camera view of hands between frontal and horizontal views
 - Special "Preferences Menu" context responses (no repeat of name response)
     - When checked: Using horizontal view of hands.
     - When unchecked: Using frontal view of hands.
     
[ ] Flip hand operation between mirrored hand control and opposite hand control
 - Special context responses (no repeat of name response)
     - When checked: Using opposite hand control.
     - When unchecked: Using mirrored hand control.

**Comment by zepumph** [comment]
>no repeat of name response

Fixed over in https://github.com/phetsims/friction/issues/289.

I am ready for a review here. I wasn't sure just how much description to add, most was delivered by common code. Let me know what you think!

**Comment by zepumph** [comment]
@terracoda and I went through this for the above commits. Please let me know if there is anything else here!

**Comment by terracoda** [comment]
Ok, here's the final wording in the dialog.
@zepumph requested that the Camera Input; Hands part have general help text.

<img width="781" alt="image" src="https://user-images.githubusercontent.com/1902116/184988236-f450f22a-06fc-4cc5-9691-cf3522ab66ac.png">
